composer.json

```diff
@@ -32,7 +36,8 @@
         "drupal/redis":"1.1.0",
```

Comment:
Adding scaffold-tooling would provide the drupal modules (https://github.com/govCMS/scaffold-tooling/blob/master/composer.json#L4-L21) that we currently provide here - so we'll need to iron out the duplication and control from one spot.